common/models/workspace.js

```diff
@@ -142,7 +163,12 @@ module.exports = function(Workspace) {
       // when merging arrays, concatenate them (lodash replaces by default)
       sources.push(function templateMergeCustomizer(a, b) {
         if (_.isArray(a)) {
-          return a.concat(b);
+          // Skip the inherit of supportedLBVersions
+          if (a.indexOf('skipInherits') > -1) {
```

Comment:
I find the "skipInherits" flag not very elegant, I am proposing a different solution: remove "supportedLBVersions" property before we add the inherited template data to the list of sources, see lines 126-134 (147-155) above.

```js
       var sources = [template];
       /* eslint-disable one-var */
       if (template.inherits) for (var ix in template.inherits) {
         var t = template.inherits[ix];
         var data = this._loadProjectTemplate(t);
         if (!data) return null; // the error was already reported
         delete data.supportedLBVersions; //<< added by me
```

Thoughts?
         sources.unshift(data);
       }
       /* eslint-enable one-var */
```